src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int anInt = random.nextInt(5);
```

Comment:
The variable name 'anInt' is not descriptive. Consider using a name that reflects its purpose, like 'figureIndex' or 'figureType'.